src/main/java/nextstep/subway/line/dto/LineResponse.java

```diff
@@ -13,7 +16,11 @@ public class LineResponse {
     private String name;
     private String color;
     private List<StationResponse> stations;
+    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
     private LocalDateTime createdDate;
+    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
```

Comment:
```suggestion
    @JsonSerialize(using = LocalDateTimeSerializer.class)
```
dto 부분을 String 으로 바꿔주거나 위와 같이 애노테이션 설정해보시겠어요?

Reply:
@JsonSerialize(using = LocalDateTimeSerializer.class)로 변경해도 동일한 에러가 발생합니다.

```
2022-05-08 18:57:50.740 ERROR 2240 --- [nio-8080-exec-1] o.a.c.c.C.[.[.[/].[dispatcherServlet]    : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is org.springframework.data.redis.serializer.SerializationException: Could not read JSON: failed to lazily initialize a collection, could not initialize proxy - no Session (through reference chain: java.util.ArrayList[0]->nextstep.subway.line.domain.Line["sections"]); nested exception is com.fasterxml.jackson.databind.JsonMappingException: failed to lazily initialize a collection, could not initialize proxy - no Session (through reference chain: java.util.ArrayList[0]->nextstep.subway.line.domain.Line["sections"])] with root cause
```